src/physics.ts

```diff
@@ -410,6 +426,9 @@ export namespace physics {
         world.frameCount++;
 
         for (const body of world.dynamicBodies) {
+            if (!body.velocity && !body.acceleration) {
```

Comment:
I'm going to merge the overall change and remove this bit, you can't determine bodies at rest as simply as if they don't currently have velocity or acceleration. They may get it next frame through penetration of other bodies being resolved.